pkg/migration/runner.go

```diff
@@ -78,6 +80,7 @@ type Runner struct {
 	table           *table.TableInfo
 	newTable        *table.TableInfo
 	checkpointTable *table.TableInfo
+	sentinelTable   *table.TableInfo
```

Comment:
Because sentinel is optional, I would prefer not to add it to the struct. We can construct sentinelName in the couple of places its needed.